src/lavinmq/http/controller/users.cr

```diff
@@ -59,6 +59,7 @@ module LavinMQ
             unless u.as_s?
               bad_request(context, "Field 'users' must be array of user names")
             end
+            @amqp_server.users.clear_permissions(u.as_s)
             @amqp_server.users.delete(u.as_s, false)
```

Comment:
Instead of remember to call `User#clear_permissions` before delete, why not call `User#clear_permissions_cache` cache from `UserStore#delete`? That would make `UserStore#clear_permissions` obsolete for now.

Reply:
I tested this, and when I delete a user the permissions are still set on a user if I try and use `clear_permissions_cache`, becasue that mehod does not actually clear the permissions.. 
If it should actually clear the permissions is another question, maybe? but maybe also permissions vs permissions cache is good to keep separate